guacamole/src/main/frontend/src/app/client/controllers/clientController.js

```diff
@@ -44,7 +44,7 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
     const requestService         = $injector.get('requestService');
     const tunnelService          = $injector.get('tunnelService');
     const userPageService        = $injector.get('userPageService');
-
+    const guacFullscreen         = $injector.get('guacFullscreen');
```

Comment:
Nitpick, but could you put this service in alphabetical order with the ones above it, and then keep the space below this before the next code comment?

Reply:
This is fairly important, IMHO. Maintaining alphabetical order for things like `#include`, `import`, blocks of retrievals from the AngularJS `$injector`, etc. helps avoid merge conflicts.

Reply:
done